src/Symfony/Bundle/FrameworkBundle/Command/CachePoolClearCommand.php

```diff
@@ -32,18 +32,14 @@
 #[AsCommand(name: 'cache:pool:clear', description: 'Clear cache pools')]
 final class CachePoolClearCommand extends Command
 {
-    private Psr6CacheClearer $poolClearer;
-    private ?array $poolNames;
-
     /**
      * @param string[]|null $poolNames
      */
-    public function __construct(Psr6CacheClearer $poolClearer, ?array $poolNames = null)
-    {
+    public function __construct(
+        private readonly Psr6CacheClearer $poolClearer,
+        private readonly ?array $poolNames = null,
```

Comment:
What is the reason for adding `readonly` here but not in other places?

Reply:
Oh, how did that happen? 😓